<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmPracticeBaseInfo</class>
 <widget class="QMainWindow" name="FrmPracticeBaseInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>560</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Xuất tệp tin</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>661</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Thông tin thực hành</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_unit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>481</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Thực hành đo lường cảm biến</string>
     </property>
     <property name="placeholderText">
      <string>Thực hành đo lường cảm biến</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>33</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Bài thực hành:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_student">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Trần văn B</string>
     </property>
     <property name="placeholderText">
      <string>Trần văn B</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>63</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Học sinh:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_class">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="placeholderText">
      <string>6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>93</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Lớp:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Nhận xét:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_description">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>481</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="plainText">
      <string>Mô tả chi tiết</string>
     </property>
     <property name="placeholderText">
      <string>Nhận xét...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>123</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Nhóm:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_group">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>A1</string>
     </property>
     <property name="placeholderText">
      <string>A1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Thông tin báo cáo</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_header">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>27</y>
       <width>481</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Báo cáo thí nghiệm</string>
     </property>
     <property name="placeholderText">
      <string>Báo cáo thí nghiệm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Môn học:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Giáo viên hướng dẫn:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_subject">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>57</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Vật lý</string>
     </property>
     <property name="placeholderText">
      <string>Vật lý</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_teacher">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>87</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Nguyễn văn A</string>
     </property>
     <property name="placeholderText">
      <string>Nguyễn văn A</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tiêu đề:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>661</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Thông số</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit_parameter">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>621</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Thông số thực hành...</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_parameter">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>501</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Xuất các thông số thực hành theo tệp</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_header</tabstop>
  <tabstop>lineEdit_subject</tabstop>
  <tabstop>lineEdit_teacher</tabstop>
  <tabstop>lineEdit_unit</tabstop>
  <tabstop>lineEdit_student</tabstop>
  <tabstop>lineEdit_class</tabstop>
  <tabstop>lineEdit_group</tabstop>
  <tabstop>plainTextEdit_description</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
